code/modules/mob/living/carbon/human/species_types/squidpeople.dm

```diff
@@ -19,19 +30,88 @@
 /datum/species/squid/random_name(gender,unique,lastname)
 	if(unique)
 		return random_unique_squid_name(genderToFind = gender)
-
 	var/randname = squid_name(gender)
-
 	if(lastname)
 		randname += " [lastname]"
-
 	return randname
 
 /proc/random_unique_squid_name(attempts_to_find_unique_name=10, genderToFind)
     for(var/i in 1 to attempts_to_find_unique_name)
         . = capitalize(squid_name(genderToFind))
         if(!findname(.))
             break
-			
+
 /datum/species/squid/after_equip_job(datum/job/J, mob/living/carbon/human/H)
-	H.grant_language(/datum/language/rlyehian)	
+	H.grant_language(/datum/language/rlyehian)
+	var/datum/action/innate/squid_change/S = new
+	S.Grant(H)
+
+/datum/species/squid/on_species_loss(mob/living/carbon/human/H)
+	fixed_mut_color = rgb(128,128,128)
+	H.update_body()
+
+/datum/action/innate/squid_change
+	name = "Color Change"
+	check_flags = AB_CHECK_CONSCIOUS
+	icon_icon = 'icons/mob/animal.dmi'
+	button_icon_state = "squid"
+	var/cooldown = 0
+	var/new_color = "FFFFFF"
+
+/datum/action/innate/squid_change/IsAvailable()
+    if(cooldown > world.time)
+        return FALSE
+    return TRUE
```

Comment:
```suggestion
    return ..()
```

Pretty sure you need this so it runs the parent's sanity checks (assuming there are any, too tired to look).